### PR TITLE
feat(hue): log format with timezone

### LINKED ISSUE #170: Hue: format logs and collect them
- Hue logs should be formatted as close to all other tdp logs
- They should be collected with promtail

### PR DESCRIPTION
**Pull request by giem-git**
#### Which issue(s) this PR fixes

Fixes #170 

#### Agreements

- [x] I hereby declare this contribution to be licensed under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0).
- [x] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

File: roles/hue/server/tasks/install.yml [hunk 1]
```diff
@@ -119,10 +119,19 @@
     group: root
     mode: "755"
 
+- name: Get current Timezone
+  command: timedatectl show --property=Timezone --value
```
Comment (kpgtek) [inline roles/hue/server/tasks/install.yml:123]: We need to use full module path: ```ansible.builtin.command```

File: tdp_vars_defaults/tdp-cluster/tdp-cluster.yml [hunk 2]
```diff
@@ -35,8 +35,12 @@ exporter_livy_spark3_http_port: 18204
 #################################
 # Service & Component logs dirs #
 #################################
+tdp_python_date_pattern: "%Y-%m-%dT%H:%M:%S.%f%z"
```
Comment (kpgtek) [inline tdp_vars_defaults/tdp-cluster/tdp-cluster.yml:38]: Will you need to add the following lines (this is done in the PR of Jupyterhub)?
```
tdp_date_iso8601_with_tz_enable: true
# If tdp_date_iso8601_with_tz_enable is false, %(asctime)s will be formatted with following pattern :
```

Comment (giem-git) [reply tdp_vars_defaults/tdp-cluster/tdp-cluster.yml:38]: Yes, this is done in the Jupyterhub PR. `tdp_date_iso8601_with_tz_enable` is not used by hue configuration. And the other line is a comment that does not make sense without the first line.

File: roles/hue/server/tasks/install.yml (hunk 1 above)
Comment (giem-git) [reply roles/hue/server/tasks/install.yml]: Done in last push

**Review by kpgtek — APPROVED** [review]

**Pull request merged**